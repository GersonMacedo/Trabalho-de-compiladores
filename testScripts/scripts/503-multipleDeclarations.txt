! -s RUN --disable-logs printer builder
! should run without errors the program with multiple declarations
program p ;
    var a : integer;
    var b : integer;
    var c : boolean;
    var d : boolean;
    var e : integer;
    var f : boolean;
    var a1 : integer;
    var aa : integer;
begin
end .