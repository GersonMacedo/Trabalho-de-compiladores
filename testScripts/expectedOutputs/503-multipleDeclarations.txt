
Performing build generation and running at testScripts/scripts/503-multipleDeclarations.txt
No errors found
Building the program

********** TAM Interpreter (Java Version 2.1) **********

Program has halted normally.
